src/qml/nodemodel.cpp

```diff
@@ -44,6 +120,13 @@ void NodeModel::initializeResult([[maybe_unused]] bool success, interfaces::Bloc
     // TODO: Handle the `success` parameter,
     setBlockTipHeight(tip_info.block_height);
     setVerificationProgress(tip_info.verification_progress);
+
+    const CBlockIndex* tip;
+    node::NodeContext* context = m_node.context();
+    ChainstateManager* chainman = context->chainman.get();
```

Comment:
this is too low level of an object to be handling here when we have `interfaces::chain`

Reply:
Let me take a look into replacing this low-level object handling with interfaces::chain